lms/resources/_js_config/__init__.py

```diff
@@ -365,19 +365,17 @@ def add_deep_linking_api(self):
         self._config["filePicker"]["deepLinkingAPI"] = config
 
     def enable_instructor_dashboard_entry_point(self, assignment):
-        self._config["hypothesisClient"] = self._hypothesis_client
         self._hypothesis_client["dashboard"] = {
             "showEntryPoint": True,
-            "entryPointRPCMethod": "openDashboard",
-        }
-        self._config["dashboard"] = {
-            "dashboardEntryPoint": {
+            "authTokenRPCMethod": "requestAuthToken",
+            "entryPoint": {
                 "path": self._request.route_path(
                     "dashboard.launch.assignment", id_=assignment.id
                 ),
-                "data": {},
             },
+            "authFieldName": "authorization",
```

Comment:
We have discussed that it probably makes sense the field name where the auth token is set is not hardcoded in the FE, but the BE can control it.

However, I'm not sure if this is the best way to define it.

Reply:
Yeah that makes sense.

Not sure if this belongs to the entryPoint part (is being used to submit to entryPoint.path) or the top level like here.

Or maybe we don't need the nesting at all?

I'm happy for you to decide depending on how it looks in the client.

Reply:
> Not sure if this belongs to the entryPoint part (is being used to submit to entryPoint.path) or the top level like here.

Yeah, I thought the same. In fact we have a `data` prop that we could use, but I reckon that is intended to pass already resolved extra params, when here we just want to know the name and the client will dynamically resolve the value later on.

On the other hand, the `path`, `data`, etc. nested props belong to a type which is defined in the LMS frontend, not the client, but I thought it might make sense keep that slightly consistent.